lib/envars.ml

```diff
@@ -12,7 +12,36 @@ open Util
 
 (** {1 Helper functions} *)
 
-let getenv_else s dft = try Sys.getenv s with Not_found -> dft ()
+let parse_env_line l =
+  try Scanf.sscanf l "%[^=]=%S" (fun name value -> Some(name,value))
+  with _ -> None
+
+let with_ic file f =
+  let ic = open_in file in
```

Comment:
Could it be that flag `Open_creat` is needed?

On MacOS X this fails in `bash` (bash 4.4) with `coq_environment.txt: No such file or directory`. (However it works with `tcsh`.)

Reply:
if the file is not there it should fail (and raise Not_found).
Are you saying that I don't catch the right exception?